### DISCUSSION TITLE
Pimcore 11 audio files - I am having problems with .wav file types.

### DISCUSSION THREAD

**Discussion by Gar1606** [pimcore/pimcore#17918] [Q&A / Support]
The website I am working on needs to provide digital samples for a digital drum. The mp3 files work fine but the wav files show up as 0:00 / 0:00 in the preview and are not playing on the website. Is there something I need to set to allow .wav files?
The upload of the files has worked because when I download the files from the admin I can play them.

**Comment by Gar1606** [comment]
@kingjia90 

Maybe you can give me a quick answer to this one please? We migrated from pimcore 4 to pimcore 11 and now I a m moving our website over to the new version. Audio files that worked in pimcoer 4 are not working in the new pimcore. .wav files do not work but mp3 do, is there something I can do?

**Reply by kingjia90** [reply]
Thank you for reporting, afaik there are no changes on that, must be some bug, could it be that MIME type is not properly recognized as it got replaced by something else over the time (symfony mime guesser)? 

It might require some local debugging, unless you have some wav that could share. 

Hope it helps

**Reply by Gar1606** [reply]
Hi, thanks for the reply.
I downloaded one of the wav files from the pimcore admin and checked it, it seems to be fine. 
I have added the file so you can take a look. I really don't know what more I can do from my side but I am open to suggestions.
[Blurred_Short.zip](https://github.com/user-attachments/files/18177311/Blurred_Short.zip)

**Comment by kingjia90** [comment]
I was able to reproduce it on the public demo, it seems that the `wav` is not whitelisted anymore by default

https://github.com/pimcore/demo/blob/08f9265940ff0769f9fc42e02ad91c6c0b050189/.docker/nginx.conf#L105
https://github.com/pimcore/skeleton/blob/5ae95060a9540251647ecfaaba9fc2f205b9fa07/.docker/nginx.conf#L105
causing it to not be found 
![image](https://github.com/user-attachments/assets/b3567e77-20d8-4062-b1f4-7901398f302b)


It works for mp3, because of `mp\d` , just add `|wav|` should fix it

**Reply by Gar1606** [reply]
unfortunately I don't have access to those config files, our system admin updated the code but it has not fixed the problem. He also removed the mp\d and the mp3 audio files still worked. He thinks it is coming from some routing, something inside pimcore but I am at a loss to figure it out. Thank you for your help.
